src/main/resources/db/migration/common/V8__Delete_message_subject_and_bodies_add_template_params.sql

```diff
@@ -0,0 +1,8 @@
+alter table message
+    drop column if exists email_body,
+    drop column if exists sms_body,
+    drop column if exists subject;
+
+alter table message
+    add template_params jsonb NULL;
```

Comment:
I'm assuming that we do not have the ability to "migrate" this as we weren't storing it in the past?

Reply:
Correct, unless we wanted to so some very intense reverse-engineering using the templates and the generated message bodies.